modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase

```diff
@@ -1,28 +1,166 @@
 @component-name = "portal-commerce"
 definition {
 
+	property custom.properties = "jsonws.web.service.paths.excludes=";
 	property portal.release = "true";
 	property portal.upstream = "true";
 	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
 	property testray.main.component.name = "Product Info Management";
 
 	setUp {
-		CommerceConfiguration.commerceSetUp(storeFront = "true");
+		CommerceConfiguration.commerceSetUp(
+			minium = "true",
+			storeFront = "true");
 	}
 
 	tearDown {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-10491. The product can be added to cart until there's stock quantity"
+	@priority = "3"
+	test CanOnlyPlaceOrderWhenStockIsAvailable {
+		property portal.acceptance = "true";
+
+		task ("Given a buyer user is created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And given the warehouse quanitity for the products is set to 4") {
```

Comment:
Please modify the word "quantity", also in the second scenario.

And only one product's quantity is being set, not "products", you should pass singular form.